### PR TITLE
Install foreman-proxy-journald when JOURNALD is set

### PR DESCRIPTION
**Pull request by lzap**
I am trying to find a place where to put spec for this but I am lost in this spec codebase. Help.

### CODE REVIEW

File: spec/acceptance/basic_spec.rb [hunk 1]
```diff
@@ -37,6 +37,10 @@ class { '::foreman_proxy':
 
   it_behaves_like 'a idempotent resource'
 
+  describe package('foreman-proxy-journald') do
+    it { is_expected.not_to be_installed }
+  end
+
```
Comment (lzap) [inline spec/acceptance/basic_spec.rb]: Saw this in `foreman` codebase, not sure if it is the correct place here.

Comment (ekohl) [reply spec/acceptance/basic_spec.rb]: It's the right place.

**Comment by mmoll** [comment]
@ekohl I *think* all should be good once https://github.com/theforeman/foreman-packaging/pull/3098 is merged.

**Comment by mmoll** [comment]
DEB PR got merged.

**Pull request merged**